extensions/lifecycle/conductor/LifecycleConductor.js

```diff
@@ -578,10 +621,13 @@ class LifecycleConductor {
                                 result.Contents.forEach(o => {
                                     marker = o.key;
                                     const [canonicalId, bucketName] = marker.split(constants.splitter);
-                                    nEnqueued += 1;
-                                    queue.push({ canonicalId, bucketName });
-
-                                    this.lastSentId = o.key;
+                                    if (!this._isBlacklisted(canonicalId, bucketName)) {
+                                        nEnqueued += 1;
+                                        queue.push({ canonicalId, bucketName });
+                                        this.lastSentId = o.key;
+                                    } else if (this.onlyBlacklistAccounts) {
+                                        marker = `${canonicalId};`;
```

Comment:
i guess this is ported from the orginal code in 7.x ; what is the purpose of this new case here?
(maybe add a comment explaining why/when we need to set a marker or not, when blacklisted)